runsc/container/container.go

```diff
@@ -1633,6 +1698,49 @@ func (c *Container) donateGoferProfileFDs(conf *config.Config, donations *donati
 	return nil
 }
 
+// Precondition: !c.shouldSpawnGofer().
+func (c *Container) prepareMounts(rootfsHint *boot.RootfsHint) ([]*os.File, *os.File, error) {
```

Comment:
Hmm I think we should not do this, instead just return a `nil` from `createGoferProcess()` for `mountsFile`. And the caller should interpret that as "just use `Spec.Mounts` as is, not cleaning required"

Reply:
Good idea! Will do.